satpy/etc/readers/abi_l1b.yaml

```diff
@@ -102,6 +102,9 @@ datasets:
       reflectance:
         standard_name: toa_bidirectional_reflectance
         units: "%"
+      counts:
+        standard_name: raw_counts
```

Comment:
Where is the name "raw_counts" from? The existing readers that have counts use `standard_name: counts`.

Reply:
Ah the standard_name from the YAML isn't actually used as far as I can tell. The calibrate method sets its own information. This should probably be updated to match that or removed.

Reply:
Agreed, we made this change Dave.